README.md

```diff
@@ -8,9 +8,19 @@ npm i
 ```
 in the repository root
 
+You also need to install rethinkdb by downloading it and running it from command prompt
+
+Please remember to insert the uservoice details into the config as well as bot token and role ids for Admins/Mods.
+
 # Run
 You can run the bot just as easliy as you can install it, just run
+
+```npm run-script dbcreate
+```
+
 ```
 npm start
 ```
 in the repository root
+
+Make sure that rethinkdb is running!
```

Comment:
If you are getting a bunch of errors in console, make sure that rethinkdb is running